alarm/utils.py

```diff
@@ -30,6 +37,26 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
                     f"threshold: {threshold}; "
                     f"threshold broken: {threshold_broken}")
         if threshold_broken:
+            ThresholdBrake.objects.create(threshold=threshold)
             return True
 
     return False
+
+
+def make_call(trade_pair):
+    threshold_prices = Threshold.objects.filter(trade_pair=trade_pair).values_list('price', flat=True)
+
+    formatted_prices_for_user_message = ', '.join([f'{price}$' for price in threshold_prices])
+    formatted_trade_pair_for_user_message = formatting_trade_pair_for_user_message(trade_pair)
+
+    last_candle = Candle.objects.filter(trade_pair=trade_pair).order_by('-modified').last()
+
+    last_candle_high_price = last_candle.high_price if last_candle else None
+    message = f"<Response><Say>Attention! Trade pair {formatted_trade_pair_for_user_message} has broken thresholds {formatted_prices_for_user_message} and current price is {last_candle_high_price}$</Say></Response>"
+    call = twilio_client.calls.create(
```

Comment:
Again, what if I want to get info about trade pair changes from a terminal? Code for that in inlined in a make call function. As I have written above, I cannot now reuse code for getting an info about threshold breaks to use it from terminal or log to a file.

Reply:
++